Gemfile

```diff
@@ -1,4 +1,9 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
 # Specify your gem's dependencies in sapience.gemspec
 gemspec
+
+gem "pry-nav"
+gem "sentry-raven"
+gem "statsd-ruby"
+gem "semantic_logger", github: "mhenrixon/semantic_logger"
```

Comment:
For starters they didn't have an appender for Sentry (Raven). I cloned their repo to create a pull request for that but I can't really use their repo until they merge it.